experiments/AMIP/user_io/debug_plots.jl

```diff
@@ -5,33 +5,54 @@ using ClimaCoupler.Interfacer: ComponentModelSimulation, SurfaceModelSimulation
 
 # plotting functions for the coupled simulation
 """
-    debug(cs::CoupledSimulation, dir = "debug")
+    debug(cs::CoupledSimulation, dir = "debug", cs_fields_0 = nothing)
```

Comment:
`cs_fields_init` might be a more clear name than `cs_fields_0`. At first I thought the 0 was a type of indexing

Reply:
Good point, but maybe let's change it to `cs_fields_ref`, as these fields may not necessarily be the initial fields.